src/api/index.tsx

```diff
@@ -76,6 +76,24 @@ export const callApi = async ({
   }
 };
 
+type GetVersionFileParams = {
+  apiState: ApiState;
+  filename?: string;
```

Comment:
so this should be `file` or `path`, no? I'd go for `file` but I already hear you saying that `path` is what's used in the rest of this patch.

Reply:
The docs for the API just say `file`, so I assumed it was `filename`, but I guess it must be `path` because `filename` might not be unique. I guess I'll use `path` to keep it consistent with the rest of the code.